webvr_ar_extension.md

```diff
@@ -54,6 +54,21 @@ struct VRHit {
 
 The `modelMatrix` property is a 4x4 transformation matrix represented as a Float32Array of size 16, encoding the position and orientation of some object in the real world.
 
+It's also possible to subscribe to events that deliver information on detected planes in the real world. The VRDisplay object is an EventTarget that fires 3 additional events ('planesadded', 'planesremoved', 'planesupdates') as response to changing in the detected planes.
+
+```
+interface VRPlane {
+  readonly attribute long identifier;
+  // A 4x4 transformation matrix, encoding the position and orientation of the plane
+  readonly attribute Float32Array? modelMatrix;
+  // Contains a values [x, z] that represent horizontal and vertical extent
```

Comment:
its not horizontal and vertical extent - it's extents relative to the plane's plane.